components/formats-bsd/src/loci/formats/tools/AmiraParameters.java

```diff
@@ -380,7 +385,11 @@ protected Map readMap() throws FormatException, IOException {
       }
 
       String key = readKey();
-      if (key.equals("")) syntaxError("Invalid key");
+      if (key.equals("")) {
+        LOGGER.warn("Syntax warning:" + row + ":" + column + ": " + "Invalid key");
```

Comment:
Not that it needs doing in this PR, but I can see this being wrapped into a helper method managed by a metadata option to choose between strict and lenient behavior.